### ISSUE TITLE
adh rc  fails in mac

### ISSUE THREAD

**Issue by Yaskier** [ApiumhubOpenSource/adh#1]
```stdExec error: Error: Command failed: /bin/sh -c docker ps -aq | awk ‘{print $1}’ | xargs -I {} --no-run-if-empty docker rm -f {}
xargs: illegal option -- -
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
            [-L number] [-n number [-x]] [-P maxprocs] [-s size]
            [utility [argument ...]]

(node:25997) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property ‘split’ of undefined
(node:25997) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by Yaskier** [comment]
the --no-run-if-empty doesn't work in mac

**Comment by ApiumhubOpenSource** [comment]
Should be fixed in 1.4.3
